atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -45,6 +45,13 @@ static synchronized void setLogArbitrator(KeyValueServiceLogArbitrator arbitrato
         logArbitrator = arbitrator;
     }
 
+    /**
+     * Returns a safe or unsafe arg corresponding to the supplied table reference, with name "tableRef".
+     */
+    public static Arg<TableReference> tableRef(TableReference tableReference) {
+        return tableRef("tableRef", tableReference);
```

Comment:
Ah, I think `table` vs `tableRef` is a product of inconsistency between devs enabling safe-logging 😄 

The two arg version was written to handle cases where you might have multiple table references being logged and would like to distinguish between them. This does exist in some features (though mainly for internal products as they aren't used directly by us - e.g. `TableTasks`). I think it's not used yet as the classes which I've added the safe params to e.g. `SpecificTableSweeper` and the slow log only execute queries over one table at a time.